extensions-core/stats/src/test/java/io/druid/query/aggregation/teststats/TestStatsTest.java

```diff
@@ -30,44 +30,42 @@
 import java.util.List;
 import java.util.Map;
 
-/**
- * Created by chunchen on 4/23/17.
- */
 public class TestStatsTest {
```

Comment:
I put the testing of these two post aggregators together under a file since it will be the most common use case.
But I can separate them too as your requested.

Reply:
Thanks.

Reply:
No problem! :-)